src/core/src/device.rs

```diff
@@ -183,7 +183,7 @@ pub trait Device<B: Backend>: Clone {
 
     ///
     fn create_buffer_view(
-        &mut self, &B::Buffer, Range<u64>
+        &mut self, &B::Buffer, Range<u64>, format::Format
```

Comment:
for images, we do `image, format, subresource`. I think it would be consistent to follow the order for buffers:
```rust
&mut self, &B::Buffer, format::Format, Range<u64>
```